<launch>

    <arg name="project" default="lio_sam"/>
    
    <node pkg="$(arg project)" type="$(arg project)_imuPreintegration"   name="$(arg project)_imuPreintegration"    output="screen" 	respawn="true"/>
    <node pkg="$(arg project)" type="$(arg project)_imageProjection"     name="$(arg project)_imageProjection"      output="screen"     respawn="true"/>
    <!-- <node pkg="$(arg project)" type="$(arg project)_rangeImage"          name="$(arg project)_rangeImage"           output="screen" 	respawn="true"/> -->
    <!-- <node pkg="$(arg project)" type="sp_demo.py"                         name="$(arg project)_sp_demo"              output="screen"     respawn="true"/> -->
    <node pkg="$(arg project)" type="$(arg project)_2Dto3D"              name="$(arg project)_2Dto3D"               output="screen"     respawn="true"/>
    <!-- <node pkg="$(arg project)" type="$(arg project)_featureExtraction"   name="$(arg project)_featureExtraction"    output="screen"     respawn="true"/> -->
    <node pkg="$(arg project)" type="$(arg project)_mapOptmization"      name="$(arg project)_mapOptmization"       output="screen"     respawn="true"/>
    
</launch>